README.md

```diff
@@ -23,3 +23,25 @@ The content of this document is discussed at the
 mailing list.
 
 [RTCWeb IETF Working Group](https://tools.ietf.org/wg/rtcweb/)
+
+### Contribution Guidelines
+Since we are a lot of people contributing to the specification, we have defined a few guidelines. Please follow them and we will be able to review your PR a lot faster when we don't have to point out style and other non-technical issues. Thank you.
+
```

Comment:
Shouldn't we also mention the membership-checking IPR bot and what you need to do to satisfy it here?